Car.cpp

```diff
@@ -1,7 +1,18 @@
 #include "Car.hpp"
 #include <iostream>
 
-void Car::turnLeft()      { std::cout << __FUNCTION__ << std::endl; }
-void Car::turnRight()     { std::cout << __FUNCTION__ << std::endl; }
-void Car::brake()         { std::cout << __FUNCTION__ << std::endl; }
-void Car::accelerate(int) { std::cout << __FUNCTION__ << std::endl; }
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+void Car::brake() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+void Car::accelerate(Speed speed) {
+    /*if(speed < 0) {
+        throw std::logic_error("Logic error: negative acceleration speed");
+    }*/
```

Comment:
you should delete the commented code. Also, you should add a new speed here and assign it to class members. It is good to compare if we don't reach maxSpeed also